src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const correctRobot = {};
+
+  for (const key in robot) {
+    const property = robot[key];
```

Comment:
IMHO, `property` is something more abstract, combining both key and value
```suggestion
    const value = robot[key];
```